getting-started/optional_installation_settings.md

```diff
@@ -0,0 +1,220 @@
+# Optional Installation Settings
+## Testing Support Setup
+
+Some of our tests we present in the guide examples utilize non-standard requirements that can be appended to your local rails helper. These additions will help provide an error free learning experience! Let's start with our gemfile to make sure we have required dependencies:
+
+{% code title="Gemfile" %}
+```ruby
+group :development, :test do
+  # Preferred testing suite which we use in our Solidus guides
+  gem 'rspec-rails'
+  # Setup 'factories' or prebuilt objects to expedite testing
+  gem 'factory_bot_rails' '~> 4.8'
+  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+end
+
+#...
+
+group :test do
+# Adds support for Capybara system testing and selenium driver
+  gem 'capybara', '>= 3.26'
+  # Easy installation and use of web drivers to run system tests with browsers
+  gem 'webdrivers'
+  # Scrubs database clean between tests
+  gem 'database_cleaner-active_record'
+```
+{% endcode %}
+
+Let's go over installation of these gems and why we are suggesting them!
+### RSpec
+RSpec is highly versatile testing framework that we utilize for Solidus and in our guide examples. To install it for your local project, simply Update your bundle and `Gemfile.lock` then install RSpec base files with the following commands in your shell:
+
+```bash
+bundle install
+rails g rspec:install
+```
+{% hint style="info" %}
+You may have to navigate to your `spec_helper.rb` located in the `spec` directory of your applications root folder and make sure that you have the following at the top of the file:
+  {% code title="spec/spec\_helper.rb" %}
+  ```ruby
+  ENV["RAILS_ENV"] ||= 'test'
+  require File.expand_path("../../config/environment", __FILE__)
```

Comment:
Shouldn't it be `require_relative`?

Reply:
This is the way it is created when running rails g rspec:install - I removed this note, however, after going back and cleaning everything up. the failure was with my _spec.rb's not containing require 'rails_helper'